app/Models/Forum/Topic.php

```diff
@@ -78,8 +79,20 @@ class Topic extends Model implements AfterCommit
     use SoftDeletes {
         restore as private origRestore;
     }
+    use WithDbCursorHelper;
 
     const DEFAULT_SORT = 'new';
+    const SORTS = [
+        'new' => [
+            // type 'timestamp' because the values are stored as integer in the database
+            ['column' => 'topic_last_post_time', 'order' => 'DESC', 'type' => 'timestamp'],
+            ['column' => 'topic_last_post_id', 'order' => 'DESC'],
```

Comment:
this destroys performance due to no usable index

Reply:
should I just change it to sort by topic time & id?

Reply:
just remove this and only rely on the post time for now

Reply:
ah I understand now. I misread as meaning both columns didn't have an index